src/test/rustdoc-json/nested.expected

```diff
@@ -41,8 +41,8 @@
       "inner": {
         "is_crate": false,
         "items": [
-          "0:7",
-          "0:4"
+          "0:4",
+          "0:7"
```

Comment:
Can you add a comment to `compare.py` saying that the error message is misleading and might be related to ordering?

Reply:
Yeah, will do